packages/sitecore-jss-react/src/enhancers/withEmptyFieldEditingComponent.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { ComponentType, forwardRef } from 'react';
+import {
+  GenericFieldValue,
+  Field,
+  isFieldValueEmpty,
+  FieldMetadata,
+} from '@sitecore-jss/sitecore-jss/layout';
+
+/**
+ * The HOC options
+ * */
+export interface WithEmptyFieldEditingComponentOptions {
+  /**
+   * the default empty field placeholder component
```

Comment:
Let's avoid using _placeholder_ naming
```suggestion
   * the default empty field component
```